Package.swift

```diff
@@ -1,4 +1,4 @@
-// swift-tools-version:5.8
+// swift-tools-version:5.9
```

Comment:
Is this needed?

Reply:
Yes - without this, required compiler flags for understanding the `package` visibility modifier aren't set, and the build will fail.

Reply:
But this package doesn't use _package_ visibility, all we're doing is optionally changing part of what we generate.

Reply:
Or does this value ripple in some way because of the plugin?  i.e. - could someone not use the plugin generating _package_ visibility, because this value come into play in their Package.swift instead of their locally specified version?

Reply:
Oh yes you're 100% right, we're not adding the `package` modifier to anything in this package. We should be fine then, I don't think anything will ripple as you suggest - I'll revert all this.